Dockerfile

```diff
@@ -1,30 +1,31 @@
-FROM ubuntu:14.04.3
+FROM alpine:3.3
 MAINTAINER Bryan Latten <latten@adobe.com>
 
 # Use in multi-phase builds, when an init process requests for the container to gracefully exit, so that it may be committed
-ENV SIGNAL_BUILD_STOP 99
-
 # Used with alternative CMD (worker.sh), leverages supervisor to maintain long-running processes
-ENV CONTAINER_ROLE=web
+ENV SIGNAL_BUILD_STOP=99 \
+    CONTAINER_ROLE=web \
+    CONF_NGINX_SITE="/etc/nginx/sites-available/default" \
+    CONF_NGINX_SERVER="/etc/nginx/nginx.conf" \
+    NOT_ROOT_USER=docker
+
+# Create an unprivileged user
+RUN adduser -D -S -H $NOT_ROOT_USER
 
 # IMPORTANT: update is *part* of the upgrade statement to ensure the latest on each build.
-# Installs pre-reqs, security updates
-RUN apt-get update && \
-    apt-get upgrade -yq && \
-    apt-get -yq install \
-        openssl \
-        ca-certificates \
-        software-properties-common \
+# Note: sed/grep replace the less performant, less functional busybox versions
+RUN apk update && \
+    apk upgrade && \
+    apk add \
+        sed \
+        grep \
         supervisor \
-        nano
-
-# Install latest nginx-stable
-RUN add-apt-repository ppa:nginx/stable -y && \
-    apt-get update -yq && \
-    apt-get install -yq nginx
+        nginx \
+    && \
+    rm -rf /var/cache/apk/*
 
 # Overlay the root filesystem from this repo
 COPY ./container/root /
 
 EXPOSE 80
-CMD ["/bin/bash", "/run.sh"]
+CMD ["/bin/sh", "/run.sh"]
```

Comment:
@bryanlatten we'll need to make sure we aren't using any bash-specific scripts anywhere in this container.